src/__tests__/tic-tac-toe.test.ts

```diff
@@ -77,4 +77,44 @@ describe('Tic-tac-toe', () => {
     game.place(0, 0);
     expect(game.winner).toEqual(undefined);
   });
+
+  it('will detect which cells caused a diagonal win', () => {
```

Comment:
It might be demonstrative to assert `game.winningVector` in a non-winning state, as well (where it should be `[]`).